tests/draft-next/contains.json

```diff
@@ -31,31 +31,6 @@
                 "data": [],
                 "valid": false
             },
-            {
-                "description": "object with property matching schema (5) is valid",
-                "data": { "a": 3, "b": 4, "c": 5 },
-                "valid": true
-            },
-            {
-                "description": "object with property matching schema (6) is valid",
-                "data": { "a": 3, "b": 4, "c": 6 },
-                "valid": true
-            },
-            {
-                "description": "object with two properties matching schema (5, 6) is valid",
-                "data": { "a": 3, "b": 4, "c": 5, "d": 6 },
-                "valid": true
-            },
-            {
-                "description": "object without properties matching schema is invalid",
-                "data": { "a": 2, "b": 3, "c": 4 },
-                "valid": false
-            },
-            {
-                "description": "empty object is invalid",
-                "data": {},
-                "valid": false
-            },
```

Comment:
Need to re-add the test for an object being `true`.

Reply:
Does line 107 not satisfy that?

Reply:
I suppose it does.